x/interchainquery/keeper/msg_server.go

```diff
@@ -59,22 +59,22 @@ func (k msgServer) SubmitQueryResponse(goCtx context.Context, msg *types.MsgSubm
 
 			tmclientstate, ok := clientState.(*tmclienttypes.ClientState)
 			if !ok {
-				k.Logger(ctx).Error("error unmarshaling client state", "cs", clientState)
+				k.Logger(ctx).Error(fmt.Sprintf("error unmarshaling client state %v", clientState))
```

Comment:
do we know if the keys get printed here? I think I've seen both cases in the logs. This is a pretty big struct so if the keys don't show up, it might make sense to print them as well

Reply:
this data type comes from a pb.go file, so I think it will print the keys!